### PR TITLE
update downloadpage

### PR DESCRIPTION
**Pull request by tanePON**
### What does this PR do?
downloadpageのデザインを変更。
・表に表示するデータの列を追加
・ホームに戻るボタンを追加
・チェックボックスの位置を１列目に変更

- [x] Documentation or TypeScript types (it's okay to leave the rest blank in this case)
- [ ] Code changes

### How did you verify your code works?

works in a container environment.

Confirmation of normal screen transition.

### CODE REVIEW

File: app/download/page.tsx [hunk 1]
```diff
@@ -30,84 +31,129 @@ export default function Download() {
...
-            </th>
-          </tr>
-        </thead>
-        <tbody className="divide-y divide-gray-100 border-t border-gray-100">
-          <tr className="hover:bg-gray-50">
-            {/* localFormDataなし */}
-            {formdata === undefined ? (
-              <th className="px-6 py-4 font-medium text-gray-900">
-                localFormDataなし
+      <div className="hoge">
```
Comment (Marukome0743) [inline app/download/page.tsx:34]: This div is useless.
Please remove.

File: app/download/page.tsx [hunk 2]
```diff
@@ -30,84 +31,129 @@ export default function Download() {
...
+              </td>
+              <td className="px-6 py-4">samplecompany</td>
+              <td className="px-6 py-4">999999</td>
+              <td className="px-6 py-4">999-999-888</td>
+              <td className="px-6 py-4">samplemail@samplemail.com</td>
+              <td className="flex justify-end gap-4 px-6 py-4 font-medium">
+                <a href="hoge">Delete</a>
+                <a href="hoge" className="text-primary-700">
+                  Edit
+                </a>
```
Comment (Marukome0743) [inline app/download/page.tsx:143]: This code is old.
Rebase your branch first.
Then request PR.

File: app/download/page.tsx [hunk 3]
```diff
@@ -30,84 +31,129 @@ export default function Download() {
...
-              <button type="button">Edit</button>
-            </td>
-          </tr>
-        </tbody>
-      </table>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+      <div>
```
Comment (Marukome0743) [inline app/download/page.tsx:149]: Remove unused `<div>`.

**Review by Marukome0743 — CHANGES_REQUESTED** [review]
Please rebase your branch before make PR because there are old codes which I fixed.

File: app/download/page.tsx (hunk 3 above)
Comment (tanePON) [reply app/download/page.tsx:149]: Deleted <div>Tags.

File: app/download/page.tsx (hunk 2 above)
Comment (tanePON) [reply app/download/page.tsx:143]: Rebased <button>tags and PR.

File: app/download/page.tsx (hunk 1 above)
Comment (tanePON) [reply app/download/page.tsx:34]: Removed div tags.

File: app/download/page.tsx [hunk 4]
```diff
@@ -43,20 +47,33 @@ export default function Download() {
...
             </th>
           </tr>
         </thead>
         <tbody className="divide-y divide-gray-100 border-t border-gray-100">
           <tr className="hover:bg-gray-50">
+            <th className="px-6 py-4">
+              <div className="flex h-5 items-center">
+                <input
+                  type="checkbox"
+                  id="example12"
```
Comment (Marukome0743) [inline app/download/page.tsx:66]: Do not name unrelated id.
Would you reset id value?
Or, delete id.

File: app/download/page.tsx [hunk 5]
```diff
@@ -33,6 +34,9 @@ export default function Download() {
       <table className="w-full border-collapse bg-white text-left text-sm text-gray-500">
         <thead className="bg-gray-50">
           <tr>
+            <th scope="col" className="px-6 py-4 font-medium text-gray-900">
+              hoge
+            </th>
```
Comment (Marukome0743) [inline app/download/page.tsx:39]:
```suggestion
            <th scope="col" className="px-6 py-4 font-medium text-gray-900">
            </th>
```
IMO, the value is not needed

File: app/download/page.tsx [hunk 6]
```diff
@@ -43,20 +47,33 @@ export default function Download() {
...
             <th scope="col" className="px-6 py-4 font-medium text-gray-900">
-              download
+              phone
+            </th>
+            <th scope="col" className="px-6 py-4 font-medium text-gray-900">
+              mail
             </th>
             <th scope="col" className="px-6 py-4 font-medium text-gray-900">
-              なんか入れられる
+              ?
```
Comment (Marukome0743) [inline app/download/page.tsx:56]:
```suggestion
              operation
```

File: app/download/page.tsx [hunk 7]
```diff
@@ -43,20 +47,33 @@ export default function Download() {
               employeeId
             </th>
             <th scope="col" className="px-6 py-4 font-medium text-gray-900">
-              download
+              phone
```
Comment (Marukome0743) [inline app/download/page.tsx:50]:
```suggestion
              phone number
```

**Review by Marukome0743 — CHANGES_REQUESTED** [review]

File: app/download/page.tsx (hunk 4 above)
Comment (tanePON) [reply app/download/page.tsx]: Deleted id.

File: app/download/page.tsx (hunk 5 above)
Comment (tanePON) [reply app/download/page.tsx]: Deleted value.
Fixed according to biome.

**Review by Marukome0743 — APPROVED** [review]

**Review by kazutan1230 — APPROVED** [review]
I approve this PR.

**Pull request merged**